suite-common/wallet-utils/src/__tests__/accountUtils.test.ts

```diff
@@ -20,7 +21,7 @@ import {
 } from '../accountUtils';
 import * as fixtures from '../__fixtures__/accountUtils';
 
-describe('account utils', () => {
+describe.only('account utils', () => {
```

Comment:
Forgotten?

Reply:
🤦 thanks for the catch 5a3ffa41c8d6125ab09a3442524316d6f674e162